packages/main/cypress/specs/DateControlsWithTimezone.cy.tsx

```diff
@@ -28,7 +28,7 @@ describe("Date related components in different timezone", () => {
 
 		cy.get("#datePickerNow")
 			.shadow()
-			.find("ui5-input")
+			.find("ui5-datetime-input")
```

Comment:
Can we search by attribute? It might have been missed in the past. It will help if we decide to test the scoping feature in the future.